content/en/docs/setup/independent/high-availability.md

```diff
@@ -65,48 +64,73 @@ needed.
+
+### Configure SSH
+
 1. Enable ssh-agent on your main device that has access to all other nodes in
-   the system by running `eval $(ssh-agent)`.
-1. Add your ssh identity to the session `ssh-add ~/.ssh/path_to_private_key`.
-1. When you ssh to any node make sure you add the `-A` flag like this `ssh -A 10.0.0.7`
-1. When using sudo on any node make sure to preserve the environment so the ssh
-   forwarding works, `sudo -E -s`.
+   the system:
+   
+     ```
+     eval $(ssh-agent)
+     ```
+
+1. Add your SSH identity to the session: 
+
+     ```
+     ssh-add ~/.ssh/path_to_private_key
+     ```
+
 1. SSH between nodes to check that the connection is working properly.
-1. Create an apiserver load balancer with a DNS resolvable name.
+
+**Notes:**
+
+- When you SSH to any node, make sure to add the `-A` flag:
+     
+  ```
+  ssh -A 10.0.0.7
+  ```
+
+- When using sudo on any node, make sure to preserve the environment so SSH
+  forwarding works:
+
+  ```
+  sudo -E -s
+  ```
+
+### Create load balancer for kube-apiserver
 
 {{< note >}}
-**Note**: There are many configurations for load balancers. We have picked one
-for this guide for the sake of an example, but be aware this is highly
-customizable.
+**Note**: There are many configurations for load balancers. The following
+example is only one option. Your cluster requirements may need a 
+different configuration.
 {{< /note >}}
 
-In a cloud environment you want to put your control planes behind a tcp
-forwarding load balancer. This load balancer will distribute traffic to all
-healthy control plane nodes in its target list. The health check to determine if
-an api server is healthy or not is a TCP check on the port the kube-apiserver is
-listening on (default is `:6443`).
-
-The load balancer must have a resolvable DNS name (we do not suggest using an IP
-address directly in a cloud environment). The load balancer must have the ability to
-communicate with all control plane nodes on the apiserver port and it must allow
-incoming traffic on its listening port.
+1. Create a kube-apiserver load balancer with a name that resolves to DNS.
 
-After this step, you have a resolvable DNS name for your load balancer.
+    In a cloud environment you should place your control plane nodes behind a TCP
+    forwarding load balancer. This load balancer distributes traffic to all
+    healthy control plane nodes in its target list. The health check for
+    an apiserver is a TCP check on the port the kube-apiserver listens on 
+    (default value `:6443`).
 
-You should add the control plane nodes to the load balancer but be aware they
-will fail the health check until the kube-apiserver is running.
+    It is not recommended to use an IP address directly in a cloud environment.
 
-## Stacked control plane nodes
+    The load balancer must be able to communicate with all control plane node
+    on the apiserver port. It must also allow incoming traffic on its
+    listening port.
 
```

Comment:
I trimmed some of the language about load balancer requirements, but it still seems unnecessarily repetitive. I might give it another go, but in the meantime if y'all could confirm that I've got the basic stuff here, that would be great.